gammapy/makers/utils.py

```diff
@@ -98,6 +98,38 @@ def _get_fov_coords(pointing, irf, geom, use_region_center=True, obstime=None):
     return coords
 
 
+def time_limit_rotation(maximum_rotation, pointing_altaz):
+    """
+    Compute the maximum time such that the FoV associated to a fixed RaDec position rotates by less than
+    'maximum_rotation' in AltAz frame. It assumes that the rotation rate at the provided pointing is a good estimate
+    of the rotation rate over the full output duration (first order approximation).
+
+
+    Parameters
+    ----------
+    maximum_rotation : `~astropy.units.Quantity`
+        Maximum rotation angle.
+    pointing_altaz : `~astropy.coordinates.SkyCoord`
+        Pointing direction.
+
+    Returns
+    -------
+    duration : `~astropy.units.Quantity`
+        Time associated with the requested rotation.
+    """
+    earth_rotation_deg_per_second = 0.004167
+    denom = (
+        earth_rotation_deg_per_second
+        * np.cos(pointing_altaz.location.lat.rad)
+        * np.cos(pointing_altaz.az.rad)
+    )
+    if denom == 0:
+        return 3600 * u.s
+    return (
+        maximum_rotation.to_value(u.deg) * np.cos(pointing_altaz.alt.rad) / denom
+    ) * u.s
```

Comment:
You could handle the unit differently by giving units to `earth_rotation_deg_per_second`.
PS: @maxnoe , is there no astropy functions that provides this number?

Reply:
I think this is just `360 * u.deg / u.day`, correct?

Reply:
yes:
```

In [2]: (360 * u.deg / u.day).to(u.deg/u.s)
Out[2]: <Quantity 0.00416667 deg / s>
```

Reply:
So I'd add a constant up to:

```
EARTH_ANGULAR_VELOCITY = (360 * u.deg / u.day).to(u.deg / u.s)
```

Reply:
Done: added a constant, keeping the unit as deg/day since it doesn't affect usage